src/views/Meme.vue

```diff
@@ -1,11 +1,90 @@
 <template>
-  <div>
-    hi
+  <div class="form-wrapper">
+    <form class="meme-form">
+      <h1>Meme Form</h1>
+      <span class="meme-title-input">
+        <input type="text" 
+               name="meme-title"
+               required />
+        <label for="meme-title">Meme Title</label>
+      </span>
+      <input type="file"
+             id="meme"
+             name="meme"
+             accept="image/png, image/jpeg" />
+      <button>Send</button>
+    </form>
   </div>
 </template>
 
 <script>
 export default {
 	
 };
-</script>
\ No newline at end of file
+</script>
+
+<style lang="scss" scoped>
```

Comment:
split the necessary scoped fields and non necessary scoped fields
if you need to edit a style edit the global one thats